DbAdapterInterface/conanfile.py

```diff
@@ -10,8 +10,7 @@ class DbAdapterInterfaceConan(ConanFile):
     topics = ("conan", "db", "sql", "adapter", "wrapper")
     license = "MIT"
     generators = "cmake_find_package"
-    settings = "os", "compiler", "build_type", "arch"
-    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0"]}
+    options = {"boost": ["1.66.0", "1.67.0", "1.72.0", "1.75.0", "1.76.0"]}
     default_options = {"boost": "1.72.0"}
```

Comment:
Why boost version by default is 1.72.0 instead lastest version?

Reply:
Yes, good point, we should be using the latest version if not provided. Fixed.